Wikimate.php

```diff
@@ -1053,7 +1054,7 @@ public function getFilename()
 	public function getInfo($refresh = false)
 	{
 		if ($refresh) { // We want to query the API
-
+			// Specify all image properties to retrieve
```

Comment:
Well, it gets far more properties than for a page, and I meant it more in the sense of "list all those that are to be retrieved, and don't list those that won't be retrieved". :) But 'relevant' is fine too.

Either way, 'image' should be 'file', so thanks for highlighting this anyway.